### PR TITLE
Admin Actve Users - create user + search bar

### PR DESCRIPTION
**Pull request by hadicheaito1**
## Description

- Allows admin to create new user from Manage Users page
- Adds the search bar functionality
## Screenshots (if appropriate):

image

### CODE REVIEW

**Comment by hadicheaito1** [comment]
After merging the most recent changes, the table seems to go 'out of bounds' if there is a long name. I tried playing around with the divs but had no luck. Any ideas?



image
image

**Comment by scouillard** [comment]
> After merging the most recent changes, the table seems to go 'out of bounds' if there is a long name. I tried playing around with the divs but had no luck. Any ideas?
> 
> image

Yeah it does that everywhere.

I wouldn't worry about that for now.

We will have to fix this eventually.

**Review by scouillard — COMMENTED** [review]
We should change this to a class (.admin-table instead of #admin-table).

And please don't forget to change it in Admin/ServerRooms as well.

As of right now it is removing the styling on Admin/ServerRooms.

**Comment by scouillard** [comment]
> > After merging the most recent changes, the table seems to go 'out of bounds' if there is a long name. I tried playing around with the divs but had no luck. Any ideas?
> > image
> 
> > Yeah it does that everywhere.
> 
> > I wouldn't worry about that for now.
> 
> > We will have to fix this eventually.
> 
> This only started happening after the changes that we discussed this morning got merged right?

No, it's been like that for a while. I think it's the "wide-background" class - not sure. It's the way we are building our layout at a low level.

File: app/controllers/api/v1/admin/users_controller.rb [hunk 1]
```diff
@@ -4,11 +4,30 @@ module Api
...
+          }.merge(user_params)) # TMP fix for presence validation of :provider
+
+          if user.save
+            render_json status: :created
+          else
+            # TODO: amir - Improve logging.
+            render_json errors: user.errors.to_a, status: :bad_request
+          end
+        end
+
```
[the remark this answers is not in the document]
Comment (farhatahmad) [reply app/controllers/api/v1/admin/users_controller.rb:27]: Ah gotcha

**Review by farhatahmad — DISMISSED** [review]

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**